src/libs/ajax/leonardo/Runtimes.ts

```diff
@@ -11,7 +11,12 @@ import {
   jsonBody,
   makeRequestRetry,
 } from 'src/libs/ajax/ajax-common';
-import { GetRuntimeItem, ListRuntimeItem } from 'src/libs/ajax/leonardo/models/runtime-models';
+import {
+  SanitizedGetRuntimeItem,
+  SanitizedListRuntimeItem,
+  sanitizeGetRuntime,
+  sanitizeListRuntime,
```

Comment:
The functions and these types should likely live in `Runtimes.ts`, with a type `export type Runtime = SanitizedListRuntimeItem | SanitizedGetRuntimeItem` being the interface outside components program against unless they need to be more specific

Reply:
cc @msilva-broad, thoughts? Would you agree with the above?

Reply:
Ah, sorry, ignore this, trying to speed through my vacation backlog and totally missed the prior conversation 😬